class.jetpack.php

```diff
@@ -1819,8 +1819,10 @@ public static function get_module( $module ) {
 	 * Like core's get_file_data implementation, but caches the result.
 	 */
 	public static function get_file_data( $file, $headers ) {
+                //Get just the filename from $file (i.e. exclude full path) so that a consistent hash is generated
+                $fileName = basename($file);
```

Comment:
Just a few style issues:
1) We don't use camel case-- let's go with $file_name
2) These lines should be indented with tabs, not spaces
3) There should be spaces around $file `basename( $file )`